py/server/deephaven/table_listener.py

```diff
@@ -215,6 +215,13 @@ def _do_locked(ug: Union[UpdateGraph, Table], f: Callable, lock_type: Literal["s
     else:
         raise ValueError(f"Unsupported lock type: lock_type={lock_type}")
 
+def _locked(ug: Union[UpdateGraph, Table], lock_type: Literal["shared", "exclusive"] = "shared") -> Generator:
```

Comment:
Both `_locked` and `_do_locked` look unused in this file.  They are both private.  Can they be removed?